### ISSUE TITLE
Pebble Time Round (chalk) Firmware install

### ISSUE THREAD

**Issue by XDjackieXD** [Freeyourgadget/Gadgetbridge#602]
When trying to flash the firmware to my Pebble Time Round, it says that the pbz file is corrupted or my hardware is incompatible. when digging through the code I found out that this happens because of the combination of
https://github.com/Freeyourgadget/Gadgetbridge/blob/master/app/src/main/java/nodomain/freeyourgadget/gadgetbridge/devices/pebble/PBWReader.java#L100
```
String platformDir = determinePlatformDir(uriHelper, platform);
if (platform.equals("chalk") && platformDir.equals("")) {
    return;
}
```
and
https://github.com/Freeyourgadget/Gadgetbridge/blob/master/app/src/main/java/nodomain/freeyourgadget/gadgetbridge/devices/pebble/PBWReader.java#L240
```
if (uriHelper.getFileName().endsWith(".pbz")) {
    return platformDir;
}
```
which is probably a regression of https://github.com/Freeyourgadget/Gadgetbridge/commit/aadde7d1cad5a2b147b6f909af8e9260b93e4ca4

Just for reference: I'm running Gadgetbridge 0.17.5 on Android Marshmallow (Cyanogenmod).

**Comment by danielegobbetti** [comment]
Unrelated side note: we do not support centering the quadrant in the display and since you are the first user I am aware of with a pebble time round I wonder if you could provide btsnoops when centering it with the official app? (in a new issue :-)

**Comment by XDjackieXD** [comment]
Sure! You just have to tell me how to create a btsnoop and what scenarios you want to have covered (only displaying a notification using the official app?).

**Comment by danielegobbetti** [comment]
You can enable the btsnoop from the development menu in the  settings. 
I believe the only feature we are missing is the centering of the screen, notifications and the like are already working :-) as far as I know there is a setting (specific to round watches) to place the displayed image exactly in the middle of the screen but I don't know where it is in the official app.

**Comment by XDjackieXD** [comment]
What exactly do you mean by center an image? I couldn't find any according setting and I also wouldn't know what it would be for

**Comment by danielegobbetti** [comment]
Screen alignment is it ( see the edit of my comment for a link)

**Comment by XDjackieXD** [comment]
Here a packet capture while doing a screen alignment although I don't think that it will be of any use as screen alignment is triggered through the settings menu on the pebble itself and doesn't need any smartphone interaction (it even works in airplane mode).
[btsnoop_hci.log.txt](https://github.com/Freeyourgadget/Gadgetbridge/files/851680/btsnoop_hci.log.txt)

**Comment by ashimokawa** [comment]
@XDjackieXD 
You were absolutely right about just everything. Thanks! :+1: 
Fixed and will be in 0.18.1.

**Comment by danielegobbetti** [comment]
@XDjackieXD thanks for providing the dump so fast. I deleted part of the thread because the functionality of screen alignment lives on watch only as you suspected and I wanted to keep the issue uncluttered. Thanks again!
